vignettes/rls.Rmd

```diff
@@ -194,26 +191,30 @@ which shells a normal user is allowed to set
 
 
 ``` r
-(policy3 <- rls_construct_policy(
-  name = "user_mod",
-  table = "passwd",
-  command = "UPDATE",
-  using = "(current_user = user_name)",
```

Comment:
We need to explain the magic of `current_user`

Reply:
Will do